File: Modul 6 Sem 1/C/PRAK601-2410817110005-MuhammadRizqanKarim.c
// PRAK601-NIM-Nama.c
#include <stdio.h>

int main() {
    int rows, cols;
    printf("Masukkan jumlah baris dan kolom: ");
    scanf("%d %d", &rows, &cols);
    int elements[rows * cols];
    printf("Masukkan elemen matriks: ");
    for (int i = 0; i < rows * cols; i++) {
        scanf("%d", &elements[i]);
    }
    for (int i = 0; i < rows; i++) {
        for (int j = 0; j < cols; j++) {
            printf("%d ", elements[i * cols + j]);
        }
        printf("\n");
    }
    return 0;
}

File: Modul 6 Sem 1/Phython/PRAK601-2410817110005-MuhammadRizqanKarim.py
# PRAK601-NIM-Nama.py
rows, cols = map(int, input("Masukkan jumlah baris dan kolom: ").split())
elements = list(map(int, input("Masukkan elemen matriks: ").split()))

matrix = [elements[i * cols:(i + 1) * cols] for i in range(rows)]
for row in matrix:
    print(' '.join(map(str, row)))
